erdpy/projects/project_base.py

```diff
@@ -11,9 +11,13 @@
 logger = logging.getLogger("Project")
 
 
+# TODO use pathlib.Path everywhere
 class Project:
+
     def __init__(self, directory):
-        self.directory = str(Path(directory).resolve())
+        self.path = Path(directory).expanduser().resolve()
+        self.directory = str(self.path)
+
```

Comment:
My `flake8` config would complain (it wants one space between methods).

Reply:
Reverted.